src/brod_client.erl

```diff
@@ -186,6 +186,9 @@ start_consumer(Client, TopicName, ConsumerConfig) ->
   case get_consumer(Client, TopicName, _Partition = 0) of
     {ok, _Pid} ->
       ok; %% already started
+    {error, client_down} ->
+      Call = {start_consumer, TopicName, ConsumerConfig},
+      safe_gen_call(Client, Call, infinity);
```

Comment:
same here, if client is down, we'll need to ensure client is started, then start consumer again.